Documentation/migration-scripts/python-scripts/repush_user_kafka_messages.py

```
@@ -0,0 +1,321 @@
+            except Exception as e:
+                log(f"❌ Error processing user {user_id}: {e}")
+                continue
+        
+        if producer:
+            producer.flush()
+
+        inner_cursor.close()
+        cursor.close()
+        conn.close()
+
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)